Automatic Validation ended with:
> 2025-06-24T05:02:13.8916204Z ##[error] Manifest Error: Failed to process installer. [InstallerUrl] Value: https://download.microsoft.com/download/ac93b367-7b17-4838-a079-c6f***7bf582/CompanyPortal-Universal-Production_x64_x86_ARM_ARM64.appxupload_Windows10_PreinstallKit.zip
 2025-06-24T05:02:13.8938922Z ##[error] Manifest Error: Failed to process installer. [InstallerUrl] Value: https://download.microsoft.com/download/ac93b367-7b17-4838-a079-c6f***7bf582/CompanyPortal-Universal-Production_x64_x86_ARM_ARM64.appxupload_Windows10_PreinstallKit.zip

(Automated response - build 1078.)

--- a/manifests/m/Microsoft/CompanyPortal/11.2.1495.0/Microsoft.CompanyPortal.locale.en-US.yaml
+++ b/manifests/m/Microsoft/CompanyPortal/11.2.1495.0/Microsoft.CompanyPortal.locale.en-US.yaml
@@ -9,7 +9,7 @@ PublisherUrl: https://www.microsoft.com/
 PublisherSupportUrl: https://support.microsoft.com/
 PrivacyUrl: https://privacy.microsoft.com/privacystatement
 Author: Microsoft Corporation
-PackageName: ms-resource://Microsoft.CompanyPortal/AppConstants/ApplicationName
+PackageName: Company Portal
 PackageUrl: https://www.microsoft.com/download/details.aspx?id=108156
 License: Proprietary
 LicenseUrl: https://www.microsoft.com/legal/terms-of-use

--- a/manifests/m/Microsoft/CompanyPortal/11.2.1495.0/Microsoft.CompanyPortal.locale.en-US.yaml
+++ b/manifests/m/Microsoft/CompanyPortal/11.2.1495.0/Microsoft.CompanyPortal.locale.en-US.yaml
@@ -9,7 +9,7 @@ PublisherUrl: https://www.microsoft.com/
 PublisherSupportUrl: https://support.microsoft.com/
 PrivacyUrl: https://privacy.microsoft.com/privacystatement
 Author: Microsoft Corporation
-PackageName: Company Portal
+PackageName: ms-resource://Microsoft.CompanyPortal/AppConstants/ApplicationName
 PackageUrl: https://www.microsoft.com/download/details.aspx?id=108156
 License: Proprietary
 LicenseUrl: https://www.microsoft.com/legal/terms-of-use

